meinberlin/apps/newsletters/views.py

```diff
@@ -120,13 +135,24 @@ def get_email_kwargs(self):
     def get_form_kwargs(self):
         kwargs = super().get_form_kwargs()
         kwargs['organisation'] = self.organisation
-        kwargs.pop('user')
+        user = self.request.user
```

Comment:
Yeah, you are right, this change seems a bit scary, but worked locally for me! So, let's try around a LOT on dev!